frontend/console/local.go

```diff
@@ -24,7 +25,7 @@ var proxyURL, _ = url.Parse("http://localhost:5173") //nolint:errcheck
 var proxy = httputil.NewSingleHostReverseProxy(proxyURL)
 
 func Server(ctx context.Context, timestamp time.Time, publicURL *url.URL, allowOrigin *url.URL) (http.Handler, error) {
-	gitRoot, ok := internal.GitRoot("").Get()
+	gitRoot, ok := internal.GitRoot(os.Getenv("FTL_DIR")).Get()
```

Comment:
If you want to use an envar, plumb it through from a Kong config rather than using getenv directly

Reply:
Ah I see, this is local.go...disregard!